src/components/Greetings.js

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getRandomGreeting } from '../redux/messageReducer';
+
+const Greenings = () => {
+  const dispatch = useDispatch();
+  const { messages } = useSelector((state) => state.messages);
+
+  useEffect(() => {
+    dispatch(getRandomGreeting());
+  }, [dispatch]);
+
+  return (
+    <div className="container">
+      <h3>{messages}</h3>
+    </div>
+  );
```

Comment:
- [ ] While trying to run your react app to display the random greetings from your backend, I got blank page. You might need to look into the `cors` in your backend for this to be able to display on another PC

![app-behave-1](https://github.com/Tekhlay/hello-react-front-end/assets/26597432/33ac34b6-8429-4828-9fa3-2892d2aad219)

![app-behave-2](https://github.com/Tekhlay/hello-react-front-end/assets/26597432/1bb9d111-b3f5-4abe-91a3-29df6ff64702)